docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -510,4 +326,8 @@ The available keys are:
 |`kubernetes.selectors.*`
 |`string`
 |Kubernetes selectors
-|===
\ No newline at end of file
+|===
+
+To learn more about how you can leverage these variables for autodiscovery refer to
+<<elastic-agent-kubernetes-autodiscovery,kubernetes autodiscovery with Elastic Agent>> for more information
+about shaping dynamic inputs for autodiscovery.
```

Comment:
```suggestion
To learn more about how you can leverage these variables for autodiscovery, refer to
<<elastic-agent-kubernetes-autodiscovery,kubernetes autodiscovery with Elastic Agent>> for more information
about shaping dynamic inputs for autodiscovery.
```

Reply:
I feel like the last part of this sentence tacks on a new idea and makes the sentence harder to parse. Can you drop either the introductory phrase ("To learn more about how you can leverage these variables for autodiscovery") or the end ("for more information about shaping dynamic inputs for autodiscovery") without losing meaning here?